### PR TITLE
Images were added to img/etherscan-voting, and the etherscan_voting.m…

### PR DESCRIPTION
**Pull request by JennyKop**
Images were added to img/etherscan-voting, and the etherscan_voting.md was changed to a new one referring to the new voting contract, after vote 178 enacted

### CODE REVIEW

**Review by rkolpakov — APPROVED** [review]
LGTM

File: docs/guides/etherscan-voting.md [hunk 1]
```diff
@@ -1,62 +1,81 @@
...
-![](/img/etherscan-voting/vote-1.png)
+## Delegating Through Etherscan
 
-- `_voteId` is the vote id from the point 2.
-- `_supports` is the flag of whether you're voting for (type `true`) or against (type `false`) the vote
-- `_executesIfDecided` is the flag to enact the vote if it could be executed right away in the tx sending the vote, `true` or `false`; from the experience of the previous votes, you may leave that as `false`
+### Assign a Delegate
 
-4. Fill in the parameters `_voteId`, `_supports` & `_executesIsDecided` and send the transaction
+1. Open the Aragon Voting contract.
```
Comment (tamtamchik) [inline docs/guides/etherscan-voting.md:66]: Assuming the initial steps have been completed, this step is redundant. It is also not present in all other flows, so I don't see a reason why it is present here. Maybe remove it?

**Review by tamtamchik — CHANGES_REQUESTED** [review]
There is an open question with a different address for the 'Aragon Voting' contract. It needs to be settled before approval.

The other comments are just suggestions; they are nice to have but not essential or blocking.

File: docs/guides/etherscan-voting.md [hunk 2]
```diff
@@ -1,62 +1,81 @@
...
-## Video guide
+This guide will walk you through how to vote, override your delegate's vote, and delegate your voting power using Etherscan. If the [Voting UI](https://vote.lido.fi/) is unavailable or you prefer to vote via Etherscan, follow these simple steps.
 
-<div style={{position:'relative',width:'100%',paddingBottom:'62.5%',height:0}}>
-   <iframe style={{position:'absolute',top:0,left:0,width:'100%',height:'100%'}} src="https://www.youtube.com/embed/5YTJgudYHs8" frameBorder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowFullScreen />
-</div>
+## Getting Started
 
-## Preparation
+Obtain the address of the Lido DAO `Aragon Voting` contract from the [Deployed Contracts](https://docs.lido.fi/deployed-contracts/#dao-contracts) page. Currently, it is [0xf165148978Fa3cE74d76043f833463c340CFB704](https://etherscan.io/address/0xf165148978Fa3cE74d76043f833463c340CFB704). Open this contract on Etherscan.
```
[the remark this answers is not in the document]
Comment (tamtamchik) [reply docs/guides/etherscan-voting.md:8]: It looks like 0xf16514 is an implementation address, probably just a typo.

[the remark this answers is not in the document]
Comment (JennyKop) [reply docs/guides/etherscan-voting.md]: Correct, thanks for checking! Changed.

[the remark this answers is not in the document]
Comment (JennyKop) [reply docs/guides/etherscan-voting.md]: Changed, thanks for sharing

File: docs/guides/etherscan-voting.md [hunk 3]
```diff
@@ -1,62 +1,81 @@
...
 
-Get the address of the Lido DAO `Aragon Voting` contract from [Deployed Contracts](/deployed-contracts/#dao-contracts) page. It should be: [0x2e59A20f205bB85a89C53f1936454680651E618e].
+## Voting
 
-Get the vote id, either from [voting ui]:
+### Step 1: Find the Voting ID
 
-![](/img/etherscan-voting/voting_ui.png)
+- Go to the **Read as Proxy** tab of the Aragon Voting contract.
```
[the remark this answers is not in the document]
Comment (JennyKop) [reply docs/guides/etherscan-voting.md:15]: Make sence

**Review by tamtamchik — APPROVED** [review]
Approved! :)

**Pull request merged**